src/read/mod.rs

```diff
@@ -120,6 +120,7 @@ where
 impl<K, V, M, S> ReadHandle<K, V, M, S>
 where
     K: Eq + Hash,
+    V: Eq + Hash,
```

Comment:
I don't see why these have been added everywhere? That seems like an unfortunate requirement to add.

Reply:
They're already requirements of the map itself - so unless I'm missing anything it only makes sense for them to be there as well, no? I think it was necessary for adding the `contains_value` helpers.

Reply:
Ah, I see, they are requirements on `WriteHandle` because we unconditionally support `retain` :'( All right, I guess it makes sense to have them here too then.

Reply:
I'm pretty sure values would have to be `Eq + Hash` anyway due to the use in `hashbag`, no?